tests/custom_hypothesis_support.py

```diff
@@ -51,6 +51,11 @@
 binary_bool_operator = hs.sampled_from(['and', 'or'])
 unary_bool_operator = hs.sampled_from(['not'])
 
+# Strategies for generating builtin type names
+annotation_types = [bool, bytearray, bytes, complex, dict, enumerate
+                    , float, frozenset, int, list, set, str, tuple]
+annotation = hs.sampled_from(annotation_types)
```

Comment:
You can clean up the code below a little if you use a hypothesis strategy `map` or something here to create a strategy that returns the name of a string.